pippo-core/src/main/java/ro/pippo/core/ParameterValue.java

```diff
@@ -236,15 +236,20 @@ public String toString(String defaultValue) {
     }
 
     public Set<String> toSet() {
-        return toSet(new HashSet<>());
+        return toSet(Collections.emptySet());
     }
 
+    /**
+     *
```

Comment:
Can you add a minimal description of method. It's a little bit strange to be empty.
Thanks!